circle.yml

```diff
@@ -3,8 +3,6 @@ dependencies:
     - ./circleci.sh install-deps
     - ./circleci.sh install-make
     - ./circleci.sh setup-repos
-  cache_directories:
-    - "~/dlang"
```

Comment:
What was the original purpose of having this cache?

Reply:
Faster build times - we use the [same for Phobos](https://github.com/dlang/phobos/blob/master/circle.yml) and DRuntime and we didn't had problems with it so far :/

Reply:
Then why is it only a problem in this repository?

Reply:
Hehe, it's actually not an issue here either. The second commit fixes the CI, this was just my initial assumption and I left the commit, because we don't save that much with the cache here anyways (`install-deps` takes 3 seconds) and a cache can always be a potential error source.